static/js/accessibility/domains.js

```diff
@@ -1,58 +1,27 @@
-$(document).ready(function () {
+$(function () {
 
-  $.get("/static/data/tables/accessibility/domains.json", function(data) {
-    renderTable(data.data);
-  });
-
-
-  var renderTable = function(data) {
-    var table = $("table").DataTable({
-
-      responsive: true,
-
-      data: data,
-
-      initComplete: Utils.searchLinks,
+  // break caching
+  var time = (new Date()).getTime();
```

Comment:
For those JS files making requests to `/static/` (the `a11y` and `customer-satisfaction` pages), a cache-breaking timestamp is passed, which avoids people needing to hard refresh upon deployment.

Reply:
Fantastic! Thank you for this!